src/styles/sidebar/components/tag-editor.scss

```diff
@@ -3,6 +3,8 @@
 @use "../../variables" as var;
 
 .tag-editor {
+  margin: 0.5em 0;
```

Comment:
The SCSS for `TagList` and `TagEditor` needs to be consolidated and DRY-ed out. Task for another iteration...